dipy/reconst/dti.py

```diff
@@ -2484,30 +2485,61 @@ def design_matrix(gtab, *, dtype=None):
 
 
 @warning_for_keywords()
-def quantize_evecs(evecs, *, odf_vertices=None):
+def quantize_evecs(evecs, *, odf_vertices=None, idx=0, n_jobs=-1, engine="serial"):
     """Find the closest orientation of an evenly distributed sphere
 
     Parameters
     ----------
     evecs : ndarray
-        Eigenvectors.
+        Eigenvectors of shape (..., 3, n_evecs)
     odf_vertices : ndarray, optional
         If None, then set vertices from symmetric362 sphere.  Otherwise use
         passed ndarray as vertices
+    idx : int, optional
+        Use idx-th eigenvector as the primary eigenvector for fiber tracking.
+        If 0, uses the largest eigenvalue.
```

Comment:
Using the 0 index for that seems counter-intuitive, e.g.
https://github.com/dipy/dipy/pull/3605/files#diff-8475f43b33a3b4095f0eb13e515f92b79430c558a4ac22b42cc9875e1d40276bR1184

Why not -1 for the largest eigenvalue?

Reply:
> Why not -1 for the largest eigenvalue?

This is a convention in the current codebase. Changing it means update all tractography algorithm and  peaks related function like `peak_from_model`